app/controllers/api/v0/events_controller.rb

```diff
@@ -6,17 +6,20 @@ def create
     render(json: error, status: error.status_code) and return if error
 
     inbound_binding.events.each do |event|
```

Comment:
Does it seem weird to have the "events" key at two levels, i.e. the params would have to look like `events: { events: [...] }`.  Maybe name the internal array `items`?